spec/api-browser-view-destroy.js

```diff
@@ -0,0 +1,31 @@
+const {app, BrowserWindow, BrowserView} = require('electron')
+
+function createWindow () {
+  // Create the browser window.
+  const mainWindow = new BrowserWindow({
+    width: 800,
+    height: 600,
+    webPreferences: {
+      nodeIntegration: true
+    },
+  })
+  const view = new BrowserView();
+  mainWindow.addBrowserView(view);
+  view.webContents.destroy()
+  setTimeout(() => view.setBounds({ x: 0, y: 0, width: 300, height: 300 }), 10000) //Asynchronous Test
+  //view.setBounds({ x: 0, y: 0, width: 300, height: 300 }) //Synchronous Test
```

Comment:
We should move this back to the synchronous test for master, since it will always work.

Reply:
We actually have a folder for these things that get automatically run in a separate process to check for "did it crash".

Check out `spec-main/fixtures/crash-cases` @mlaurencin  👍

Reply:
Awesome!

Reply:
> We should move this back to the synchronous test for master, since it will always work.

Alright, I replaced the asynchronous test with the synchronous one

Reply:
For a test of this nature, we have existing spec files which encompass similar functionality - as such we should look to run it there, as part of the test suite, instead of as its own file.

[This test](https://github.com/electron/electron/blob/c8a0b2b71d8622bbb848e364a842ea12bd5966fb/spec-main/api-browser-view-spec.ts#L145) provides an example of what we'd want to aim for - since only 3-4 lines are required to trigger the crash, we shouldn't need to spin up a bespoke file

Reply:
Now that the other file has been added, this one should be deleted